app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,12 @@
+class CommentsController < ApplicationController
+  def create
+    @article = Article.find(params[:article_id])
+    @article.comments.create(comment_params)
```

Comment:
Always use `Comment.create(comment_params.merge(article_id: @article.id))`.
This one helpful when you try to find the comment creation